docs/development/extensions-core/s3.md

```diff
@@ -63,8 +63,9 @@ In addition to this you need to set additional configuration, specific for [deep
 
 ### S3 authentication methods
 
-Druid uses the following credentials provider chain to connect to your S3 bucket (whether a deep storage bucket or source bucket).
-**Note :** *You can override the default credentials provider chain for connecting to source bucket by specifying an access key and secret key using [Properties Object](../../ingestion/native-batch-input-source.md#s3-input-source) parameters in the ingestionSpec.*
+You can provide credentials to connect to S3 in a number of ways, whether for [deep storage](#deep-storage) or as an [ingestion source](#reading-data-from-s3).
+
+There is a defined order of precedence, as given below.  This means, for example, if you would like to use profile information given in `~/.aws.credentials`, do not set `druid.s3.accessKey` and `druid.s3.secretKey` in your Druid config file as this would take precedence.
```

Comment:
```suggestion
Thee configuration options are listed in order of precedence.  For example, if you would like to use profile information given in `~/.aws.credentials`, do not set `druid.s3.accessKey` and `druid.s3.secretKey` in your Druid config file because they would take precedence.
```